app_feup/lib/redux/actionCreators.dart

```diff
@@ -83,8 +103,11 @@ ThunkAction<AppState> getUserSchedule(Completer<Null> action) {
 
       action.complete();
 
-      store.dispatch(new SetScheduleAction(lectures));
-    }
+    // Updates local database according to the information fetched -- Lectures
+    AppLecturesDatabase db = await AppLecturesDatabase();
+    db.saveNewLectures(lectures);
+
+    store.dispatch(new SetScheduleAction(lectures));
```

Comment:
I see you are saving the lectures to the database on getUserSchedule. Should that not be happening on getUserExams too?

Reply:
Indeed it should. It is fixed in commit [d32f67308f2551a2c0dbacc799d5b0769721bb7a](https://github.com/NIAEFEUP/project-schrodinger/commit/d32f67308f2551a2c0dbacc799d5b0769721bb7a).

Reply:
Nice!